eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -132,6 +91,38 @@ public void exec() throws IOException {
         );
     }
 
+    /**
+     * Get objectionary from the map by given hash.
+     * @param hash Hash.
+     * @return Objectionary by given hash.
+     */
+    private Objectionary objectionaryBy(final CommitHash hash) {
```

Comment:
@maxonfjvipon Maybe `objectionary` would be more suitable name here? 
https://www.yegor256.com/2018/08/22/builders-and-manipulators.html

Reply:
@volodya-lombrozo I get qulice warning: "Field objectionary has the same name as a method". I think `objectionaryBy` is a quite good name. It's still a noun, but with preposition, so we can read it as english sentence: objectionary by hash

Reply:
@maxonfjvipon Maybe it's better to make it complete then? For example: `objectionaryByHash()`. Like in `TranspileTojos.java` method `findByOptimized()`.
What if I call your method like this: `this.objectionaryBy(tag)`. Did it give me an objectionary by tag?

Reply:
@Graur sounds better